ansible/configs/ibm-rhoic/destroy_env.yml

```diff
@@ -58,212 +58,151 @@
       set_fact:
         sandbox_master_api_key: "{{ r_sandbox_account['json']['master_api_key']['S'] }}"
 
-    - name: Get bearer token
+    ## Authenticating
+    - name: Authorize to get bearer access and refresh tokens
       uri:
         url: "{{ ibm_cloud_api_iam_url }}"
+        headers:
+          Authorization: Basic Yng6Yng=
```

Comment:
This is from IBM Cloud's documentation, not a secret we need to protect. This is username bx and password bx which is some kind of a system account in IBM Cloud. If I don't pass this header then it won't generate a refresh token which is needed for IBM Schematics plan and apply.